### PR TITLE
Update index.md

### PR DESCRIPTION
**Pull request by aJazzyFeel**
Expanded description, created section title Uses, added links to YouTube videos in More Information section, embedded image to visualize a tor network.

### CODE REVIEW

File: src/pages/security/tor/index.md [hunk 1]
```diff
@@ -1,20 +1,24 @@
...
 ---
 ## Tor
-Tor stands for "The Onion Router", which is a free software that helps anonymize its users Internet usage, maintained by the Tor Project.
-In addition to the Onion Router, the Tor Project maintains a bundle of software, most notably a web browser, which is often also
+Tor stands for "The Onion Router", which is a free software that helps anonymize its users' internet usage, maintained by the Tor Project.
+In addition to the router network, the Tor Project maintains a bundle of software, most notably a web browser, which is often also
 called Tor.
 
-The Onion Network works by bouncing a user's requests through a large worldwide, volunteer network of computers. It sends the requests through the system encrypted, only decrypting them when they get to the 'exit node', the last computer in the network for any given request before the request is sent to its destination. This makes the message look like its coming from somewhere other than where it actually originated.
+<img src="https://cdn.arstechnica.net/wp-content/uploads/2014/01/tor-structure.jpg">
```
Comment (jp-sauve) [inline src/pages/security/tor/index.md]: This should be markdown, and not a hot-linked image.
https://github.com/freeCodeCamp/guides#adding-images-to-articles

File: src/pages/security/tor/index.md [hunk 2]
```diff
@@ -1,20 +1,24 @@
 ---
-title: Tor (The Onion Router)
+title: Tor, The Onion Router
```
Comment (jp-sauve) [inline src/pages/security/tor/index.md]: I'm not sure whether this change is needed. The style guide suggests using the full name first, with the acronym in brackets.

Comment (jp-sauve) [reply src/pages/security/tor/index.md]: @fvcproductions what do you think of this?

Comment (fvcproductions) [reply src/pages/security/tor/index.md]: @jp-sauve I think should probably be changed to `Tor` as per style guidelines. 👍 

Also, we should make sure the changes in this PR are significant/substantial enough to merit a merge.

**Comment by johnkennedy9147** [comment]
closing PR as stale

**Pull request closed without merging**